enrolments/tests/test_validators.py

```diff
@@ -52,6 +53,73 @@ def test_validate_classes_in_session(self):
             session_other,
         )
 
+    def test_validate_enrolment_in_session(self):
+        session1 = Session.objects.create(season=Session.SPRING, year="2020")
+        session2 = Session.objects.create(season=Session.FALL, year="2019")
+        class1_in_session1 = Class.objects.create(
+            name="Class 1 in Session 1",
+            session=session1,
+            facilitator=self.user,
+            attendance={},
+        )
+        class2_in_session1 = Class.objects.create(
+            name="Class 2 in Session 1",
+            session=session1,
+            facilitator=self.user,
+            attendance={},
+        )
+        class1_in_session2 = Class.objects.create(
+            name="Class 1 in Session 2",
+            session=session2,
+            facilitator=self.user,
+            attendance={},
+        )
+        family1 = Family.objects.create(
+            email="fam1@test.com",
+            cell_number="123456789",
+            address="1 Fam St",
+            preferred_comms="email",
+        )
+        family2 = Family.objects.create(
+            email="fam2@test.com",
+            cell_number="123456789",
+            address="2 Fam St",
+            preferred_comms="email",
+        )
+        enrolment1_for_family1 = Enrolment.objects.create(
+            active=True,
+            family=family1,
```

Comment:
```suggestion
            family=self.family,
```
could be used if u decide not to create family1